### DISCUSSION TITLE
Is this still maintained?

### DISCUSSION THREAD

**Discussion by dylandhall** [zidoro/pomatez#670] [General]
I have a feature I'd like to add - I'd be happy to do it myself but if this repo is abandoned (it doesn't seem active?) I might look at other options - nothing worse than doing the work then never having your PR merged!

Feature is, a task in the priority task list that is a placeholder for another task list, eg a project that has a number of sub-tasks, and you want to be able to essentially say that you have two tasks in your priority list, then the next task is a placeholder for tasks in the project task list, and then further tasks in the priority list will be selected once all tasks in the project are complete.

I'm capable of adding it but I just want to know this software is still being developed..

**Comment by sekwah41** [comment]
I’m still around maintaining it. I’ve just been a bit busy with other projects e.g. I’m writing code for a charity event atm.

**Reply by neiltheneil** [reply]
Hi there! Just wanted to let you know that Pomatez is excellent! I've been using it on and off for a few years and have really gotten back to making it a daily habit. Thank you so much for creating this wonderful app; the UI is excellent, the options and features are fantastic, and your continued support is so appreciated. I'm going to share this app in some discord servers, so others may also benefit from it. Best of luck on the charity project! They are lucky to have your skills and artfulness.
